markdown.dtx

```diff
@@ -15611,6 +15706,27 @@ function M.writer.new(options)
     local name = util.cache(options.cacheDir, s, nil, nil, ".verbatim")
     return {"\\markdownRendererInputFencedCode{",name,"}{",i,"}"}
   end
+  function self.jekyllData(_, data)
+    if not self.is_writing then return "" end
+    local buffer = {}
+    table.insert(buffer,"\\mardownRendererSetJekyllDataBegin")
+    for _, v in ipairs(data) do
+      if type(v) == "table" then
+        table.insert(buffer,"\\mardownRendererSetJekyllDataSequenceBegin{"..v[1].."}{"..#v[2].."}")
+        for i,w in ipairs (v[2]) do
+          s = string.gsub(w, '%s*([^:]+):%s([^\r\n]+)[\r\n]+', '\\markdownRendererSetJekyllDataString{%1}{%2}')
```

Comment:
It would be nice to word-wrap these a bit, so that they fit into the documentation, see [markdown.pdf](https://github.com/Witiko/markdown/suites/3620723994/artifacts/87551356), page 114.